pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.targomo</groupId>
     <artifactId>java-client</artifactId>
-    <version>0.33.0-SNAPSHOT</version>
+    <version>0.33.0-SNAPSHOT-LONG</version>
```

Comment:
do we need the separate version?

Reply:
Not really, but I wanted to keep it just before the merge so it'll not break other branches on core and stats. I'll update those when all the MRs are ready to be merged in.